pantab/tests/test_roundtrip.py

```diff
@@ -1,18 +1,32 @@
-import numpy as np
 from pathlib import Path
+
+import numpy as np
 import pandas as pd
 import pandas.testing as tm
 import pytest
-from tableauhyperapi import Connection, TableName, HyperProcess, Telemetry, CreateMode
+from tableauhyperapi import Connection, CreateMode, HyperProcess, TableName, Telemetry
 
 import pantab
 import pantab._compat as compat
 
 
-def assert_roundtrip_equal(result, expected):
+def assert_roundtrip_equal(result, expected, use_float_na=False):
```

Comment:
Rather than adding this as a keyword here can you make the `expected` result explicit in the appropriate tests?